src/assets/styles/readme.md

```diff
@@ -41,22 +50,29 @@ CSS ouput:
 }
 ```
 
-#### Font-weight
+#### Font weight
+
+Two font weights are available using CSS vars.
 
-| Weight  | Value |
-| ------- | ----- |
-| Light   | 300   |
-| Regular | 400   |
-| Medium  | 500   |
-| Demi    | 600   |
+| Weight                          | Value |
+| ------------------------------- | ----- |
+| --calcite-ui-text-weight-light  | 300   |
+| --calcite-ui-text-weight        | 400   |
+| --calcite-ui-text-weight-medium | 500   |
+| --calcite-ui-text-weight-demi   | 600   |
```

Comment:
demi is declared as `700` we did this long ago in Calcite Web so that typical `<b>` and `<strong>` tags would work as expected with 0 CSS

Reply:
Cool. I was basing this on what was already in the readme.
I'll update the value here an in the SCSS.

Reply:
Updated.